code/modules/surgery/organs/lungs.dm

```diff
@@ -585,6 +585,10 @@
 	. = ..()
 
 	var/datum/gas_mixture/immutable/planetary/mix = SSair.planetary[LAVALAND_DEFAULT_ATMOS]
+
+	if(!mix?.total_moles()) // this typically means we didn't load lavaland, like if we're using #define LOWMEMORYMODE
```

Comment:
IDK about this, LAVALAND_DEFAULT_ATMOS having no mix is an error condition I do want to catch in CI

Reply:
Actually, the ashwalker breath unit test would still fail, s chill